schemas/com.snowplowanalytics.mobile/application_lifecycle/jsonschema/1-0-0

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Entity that indicates the visibility state of the app (foreground, background)",
+  "self": {
+    "vendor": "com.snowplowanalytics.mobile",
+    "name": "application_lifecycle",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+
+  "type": "object",
+  "properties": {
+    "isVisible": {
+      "description": "Indicates if the app is in foreground state (true) or background state (false)",
+      "type": "boolean",
+      "default": true
```

Comment:
I certainly don't have anything against `default`, but just in case - it won't be used anywhere (Iglu doesn't support it explicitly). Thus tables will be generated as `NOT NULL` and if user doesn't provide it - the loading will fail. Some more context:

* https://github.com/snowplow/iglu/issues/248
* https://github.com/snowplow/iglu-scala-client/issues/76

We can use it as purely information, although I afraid it encourages bad behavior.

Reply:
I agree 👍  I don't even know where I got it. That value is always set in the tracker so no need to specify this here.